contrib/cirrus/runner.sh

```diff
@@ -210,7 +210,8 @@ function _run_release() {
     bin/podman info
 
     msg "Checking podman release (or potential release) criteria."
-    dev=$(bin/podman info |& grep -- -dev)
+    # We're running under 'set -eo pipefail'; make sure this statement passes
+    dev=$(bin/podman info |& grep -- -dev || echo -n '')
```

Comment:
```suggestion
    dev=$(bin/podman info |& grep -- -dev ||  : )
```

Reply:
maybe this will be better

Reply:
@zhangguanzhang thank you, why do you prefer that form? I chose `echo -n ''` because the expression is intended as a string capture, and I wanted to make it easily clear to a maintainer that the fallthrough is an empty string. `true` is logically equivalent, but could make a future maintainer stop to think it through. (At least, that was my thinking earlier today, and I still think it holds). What do you like about the `true` form?

Reply:
agree with you